src/App/App.scss

```diff
@@ -5,9 +5,10 @@
 	@include position(absolute, 0, 0, 0, 0);
 	width: 100%;
 	height: 100%;
-	min-height: 700px;
-	min-width: 1280px;
+	min-height: 640px;
+	min-width: 480px;
 	font-size: 1.6rem;
+	@include flex(row,flex-end, stretch);
```

Comment:
It is common the practise to put sass mixins on the top of a class ruleset.
So everything bellow will override explicitly override any rules from mixin